libc/src/threads/linux/cnd_broadcast.cpp

```diff
@@ -6,16 +6,20 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include "CndVar.h"
-
 #include "src/threads/cnd_broadcast.h"
 #include "src/__support/common.h"
+#include "src/__support/threads/CndVar.h"
+
+#include <threads.h> // cnd_t, thrd_error, thrd_success
```

Comment:
probably file a bug to create a proxy header for `threads.h`

Reply:
done in 0267c00503a7